community/primitive-collections/src/main/java/org/neo4j/collection/primitive/PrimitiveIntCollections.java

```diff
@@ -750,4 +754,50 @@ public static PrimitiveIntSet asSet( int[] values )
         }
         return set;
     }
+
+    /**
+     * Adds all the items in {@code iterator} to {@code collection}.
+     * @param <C> the type of {@link Collection} to add to items to.
+     * @param iterator the {@link Iterator} to grab the items from.
+     * @param collection the {@link Collection} to add the items to.
+     * @return the {@code collection} which was passed in, now filled
+     * with the items from {@code iterator}.
+     */
+    public static <C extends Collection<Integer>> C addToCollection( PrimitiveIntIterator iterator, C collection )
+    {
+        while ( iterator.hasNext() )
+        {
+            collection.add( iterator.next() );
+        }
+        return collection;
+    }
+
+    public static List<Integer> asBoxedList( PrimitiveIntIterator iterator )
```

Comment:
Lots of moved methods now have `Boxed` as part of their names, do not think it's needed since its obvious that values will be boxed, but instead that gives an impression that you will have special kind of list that is boxed and you need go to implementation and check, what is in the box.
So would prefer them to have plain names like `asList`